Doc/using/configure.rst

```diff
@@ -835,22 +830,6 @@ Libraries options
    Build the :mod:`!pyexpat` module using an installed ``expat`` library
    (default is no).
 
-.. option:: --with-system-libmpdec
-
-   Build the ``_decimal`` extension module using an installed ``mpdecimal``
-   library, see the :mod:`decimal` module (default is yes).
-
-   .. versionadded:: 3.3
-
-   .. versionchanged:: 3.13
-      Default to using the installed ``mpdecimal`` library.
-
-   .. deprecated-removed:: 3.13 3.15
-      A copy of the ``mpdecimal`` library sources will no longer be distributed
-      with Python 3.15.
-
-   .. seealso:: :option:`LIBMPDEC_CFLAGS` and :option:`LIBMPDEC_LIBS`.
```

Comment:
I wonder if it's worth keeping a mention of `--with-system-libmpdec`, just for the purpose of saying it's been removed? Unsure though.

Reply:
Is this something that expected for removed stuff?

I see one removed option, ``--with-wctype-functions`` - it's only in whatsnew.